Jellyfin.Plugin.Webhook/Notifiers/SessionStartNotifier.cs

```diff
@@ -10,10 +14,12 @@ namespace Jellyfin.Plugin.Webhook.Notifiers;
 /// <summary>
 /// Session start notifier.
 /// </summary>
-public class SessionStartNotifier : IEventConsumer<SessionStartedEventArgs>
+public class SessionStartNotifier : IEventConsumer<SessionStartedEventArgs>, IDisposable
 {
     private readonly IServerApplicationHost _applicationHost;
     private readonly IWebhookSender _webhookSender;
+    private static readonly ConcurrentDictionary<string, DateTime> _recentEvents = new();
+    private Timer _cleanupTimer;
```

Comment:
This timer will be created every time the `SessionStartedEventArgs` event is triggered and will never execute since the object never lasts 30 minutes.